benchexec/tools/jcwit.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.result as result
+from benchexec.tools.template import ToolNotFoundException
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+
+    """
+    Tool info for jcwit
+    (https://github.com/Chriszai/JCWIT).
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("jcwit.py")
+
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def name(self):
+        return "jcwit"
+
+    def cmdline(self, executable, options, task, rlimits):
+        return [executable] + options + list(task.input_files)

```

Comment:
Does your tool actually support tasks with more than one input file? If not, I would recommend to use `task.single_input_file`.

Is is correct that your tool does not need to know the data model of the C programs it gets? Typically this is required in order to be able to interpret the program semantics correctly.